python/lsst/daf/butler/core/_butlerUri/http.py

```diff
@@ -340,9 +343,10 @@ def read(self, size: int = -1) -> bytes:
             The number of bytes to read. Negative or omitted indicates
             that all data should be read.
         """
-        log.debug("Reading from remote resource: %s", self.geturl())
```

Comment:
You don't think it could still be useful to know what the URL is, separate from timing?

Reply:
I was vacillating on this. I know that the timer message will come out regardless so I thought that seeing log output of:
```
DEBUG reading URI X
DEBUG read URI X took Y seconds
```
was redundant. Sounds like you'd prefer the log message showing the intent as well as the log message with the result.

Reply:
I suppose one argument in favor of the duplication is that they are in fact different loggers so can be configured with different levels.

Reply:
> different loggers

Exactly.